core/core-impl/src/main/java/ai/timefold/solver/core/impl/domain/solution/cloner/gizmo/GizmoSolutionClonerImplementor.java

```diff
@@ -100,14 +101,20 @@ public static Comparator<Class<?>> getInstanceOfComparator(Set<Class<?>> deepClo
                 .thenComparing(Class::getName).reversed();
     }
 
+    protected void createFields(ClassCreator classCreator) {
+        classCreator.getFieldCreator(FALLBACK_CLONER, FieldAccessingSolutionCloner.class)
+                .setModifiers(Modifier.PRIVATE | Modifier.STATIC);
+    }
```

Comment:
Could have stayed static.

Reply:
No, this method is overridden in Quarkus to do nothing, which make it a mandatory to be virtual.